configure.ac

```diff
@@ -78,6 +78,25 @@ fi
 
 AM_CONDITIONAL(HAVE_SYSTEMD, [test "$have_systemd" = "yes"])
 
+AC_ARG_WITH(elogind,
+            AS_HELP_STRING([--with-elogind],
+            [Use libelogind instead of libsystemd-login]),
+            [with_elogind=$withval], [with_elogind=no])
```

Comment:
This should use `AC_ARG_ENABLE` for consistency with `--enable-systemd`

Reply:
Replaced as suggested.